fluent-syntax/tests/serializer_fixtures.rs

```diff
@@ -57,15 +45,33 @@ fn roundtrip_unnormalized_fixtures() {
 
         let content = fs::read_to_string(&path).expect("Failed to read file");
         let parsed = parse(content.as_str()).unwrap_or_else(|(res, _)| res);
-        let parsed_without_junk = clone_without_junk(&parsed);
+
         let reserialized = serialize_with_options(&parsed, Options { with_junk: true });
         let reserialized_without_junk =
             serialize_with_options(&parsed, Options { with_junk: false });
+
+        let formatted_path = path.parent().unwrap().join("formatted");
+
+        let formatted_content = fs::read_to_string(
+            formatted_path
+                .join("junked")
+                .join(path.file_name().unwrap()),
+        )
+        .unwrap();
+        assert_eq!(formatted_content, reserialized);
+        let formatted = parse(formatted_content.as_str()).unwrap_or_else(|(res, _)| res);
+
+        let formatted_content_without_junk =
+            fs::read_to_string(formatted_path.join(path.file_name().unwrap())).unwrap();
+        assert_eq!(formatted_content_without_junk, reserialized_without_junk);
+        let formatted_without_junk =
+            parse(formatted_content_without_junk.as_str()).unwrap_or_else(|(res, _)| res);
```

Comment:
The more I fiddle with it trying to figure out what's going on with tests the more I think this approach to having expanded files for just the serialized message is not the right approach to testing. We should be able to test the original JSON fixtures and just use a query engine of some kind to extract the message bit and work around having serialized position data in our actual results.

With all these expanded and multiple entries in different formats in becomes much harder to audit what tests are actually doing and compare with other implementations.